openshift-environment-driver/src/main/java/org/jboss/pnc/environment/monitor/PullingMonitor.java

```diff
@@ -54,14 +54,23 @@
 
     private ConcurrentSet<RunningTask> runningTasks;
 
+    @Inject
+    private Configuration configuration;
+
     public PullingMonitor() {
+    }
+
+    @Inject
+    public PullingMonitor(Configuration configuration) {
+        this.configuration = configuration;
```

Comment:
There already is a `monitor`method that takes timeout as a parameter. There should be no need to make this class dependent on the configuration.